autonomi/src/client/mod.rs

```diff
@@ -7,16 +7,22 @@
 // permissions and limitations relating to use of the SAFE Network Software.
 
 pub mod address;
+pub mod error;
 
+#[cfg(feature = "data")]
+pub mod archive;
 #[cfg(feature = "data")]
 pub mod data;
-#[cfg(feature = "files")]
+#[cfg(feature = "fs")]
 pub mod files;
```

Comment:
Maybe we could rename the `files` module to `fs`? As everything in there is related to the local file system.

Reply:
Make sense